lib/screens/explore_screens/regions_screens/regions_main/components/region_event_card/region_event_card.dart

```diff
@@ -85,7 +85,7 @@ class RegionEventCard extends StatelessWidget {
                               style: Theme.of(context).textTheme.buttonLowEmphasis,
                               overflow: TextOverflow.ellipsis,
                             ),
-                            Text(DateFormat.yMMMMd().format(DateTime.parse(event.createdTime.toDate().toString()))),
```

Comment:
I do not know why I put the date of creation of the event but I think the intention is to show the start date of the event which is what matters to the user.